### ISSUE TITLE
-8 tenor clef is not transposed when converted to MIDI

### ISSUE THREAD

**Issue by rualark** [paulrosen/abcjs#401]
✅  Visual editor issue status: non-blocking (currently workaround is used: notes are transposed when rendering for MIDI)

How to reproduce: take this score
```
%%barnumbers 1
M:4/4
K:F
L:1/16
V: V0 clef=treble name="Voice 1 [Sop.] (sp. 5)"
V: V1 clef=treble name="Voice 2 [Sop.] (sp. 5)"
V: V2 clef=treble name="Voice 3 [Alt.] (sp. 5)"
V: V3 clef=treble name="Voice 4 [Alt.] (sp. 5)"
V: V4 clef=treble-8 name="Voice 5 [Ten.] (sp. 5)"
V: V5 clef=treble-8 name="Voice 6 [Ten.] (sp. 5)"
V: V6 clef=bass name="Voice 7 [Bas.] (sp. 5)"
V: V7 clef=bass name="Voice 8 [Bas.] (c.f.)"
[V: V0]z16|"^?"z16|"^?"z16|"^?"z4"^?"a4"^?"b4a4|g4f4e4d4|a12e4|d4"^?"e4f8|g8a4g4|f4e4d4c4|"^?"g8f4g4|a16|
[V: V1]z16|"^?"z4c4d4e4|f8g8|f12e4|d8g8|c8d4e4|A8a8|g4f4e4d4|c8d8|e4d4c4B4|A16|
[V: V2]z16|"^?"z8E8|D4E4F4G4|"^!"A4B4c8-|c8B8|A4B4A4G4|F4E4D8|d16|A4G4F8|"^?"B8A4"^?"B4|F16|
[V: V3]z16|"^?"z16|"^?"z8"^?"F8|C4D4E4F4|G8F4E2D2|C12D2E2|"^!"F16|"^!"D12E4|F8A,8|"^?"B,8C4D2"^?"E2|F16|
[V: V4]z4f4g4f4|e4d4c8-|c8B8|A16|d12G4|A8B4"^!"c4|"^!"d8c8|B12A2B2|c16|"^?"G4A4"^!"B8|c16|
[V: V5]z8c8-|c4B4A4G4|F16-|F12c4|B4A4G4"^!"F4|E12C4|F8E4D4|G8D8|A12G2F2|"^?"E16|"^?"A16|
[V: V6]z16|"^?"z16|"^?"z4"^?"F,4E,4D,4|C,12B,,2C,2|D,16|E,4D,4C,4"^!"B,,4|A,,12B,,2C,2|D,4E,4F,4G,4|C,4D,4C,8|"^?"B,,8G,,8|C,16|
[V: V7]"_I"F,,16|"_V"C,16|"_IV6"D,16|"_I6"A,,16|"_II6"B,,16|"_III6"C,16|"_VI6"F,,16|"_II6"B,,16|"_I6"A,,16|"_VII6"G,,16|"_I"F,,16|

```

Export MIDI.

Expected result: Tenor-8 clef should result in G3-G4 range in upper tenor voice. Intead, G4-G5 is exported. 

Solution: It seems that octave should be subtracted.

🌴  This problem can also affect all +8 and -8 clefs.

**Comment by ThorsenRune** [comment]
Hi, I like your music. Just my 2cents, I don't get the idea behind treble-8 so I prefer to transpose.  I have hacked into the transpose and split it into score=0 pitch=-8, score is for the written representation and the pitch is for the acoustic representation. I believe its clearrr and more logically.

**Comment by rualark** [comment]
Hi. Good, this isnan example from Noel-Gallon book.
If you do this, then graphical representation will not match sound. This can be acceptable in some situations, but not in music education software.

**Comment by paulrosen** [comment]
This will be in 6.0.0-beta.10

**Comment by rualark** [comment]
Cool! I will test it when MuseScore gaps are implemented.

**Comment by rualark** [comment]
Seems not fixed in current repository version, or am I missing something?

**Comment by rualark** [comment]
I confirm this issue being fixed in 6.0.0 beta.12!
